proxy/schemas/responses.py

```
@@ -133,12 +155,28 @@ def validate_input_or_messages(cls, v, info):
 class Response(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)